src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,137 @@
+@import "../utils/variables";
+
+.card {
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  box-sizing: content-box;
+  transition: transform 0.3s ease;
+
+  &__image {
+    margin: 32px 19px 40px;
+    height: 134px;
+  }
+
+  &__title {
+    width: 166px;
+    color: $Main-accent;
```

Comment:
Typically it's `main-accent` Look at [sass documentation](https://sass-lang.com/documentation/variables/)